examples/idv/index.js

```diff
@@ -24,6 +24,7 @@ app.use(session({
 const router = express.Router();
 
 router.get('/', controllers.indexController);
+router.get('/fc', controllers.indexControllerWithFaceComparison);
```

Comment:
Why `index`?

The other routes are not using that naming minus the actual index with is `/`

Reply:
You are right, this was only for checking quickly.
Before merging, we'll check with Dhivya how we want the examples to be.
The current example works with the index directly generating the session... which does not give room for configuring the session... so, yeah, maybe we should refactor so from the index you can configure the session, and therefore having only one route?